src/Message.js

```diff
@@ -28,11 +28,34 @@ export const MessageTime = ({ time, isBot }: MessageTimeProps) => {
 
 type MessageProps = {
   chat: ChatMessage,
-  onButtonClick?: (title: string, payload: string) => void
+  onButtonClick?: (title: string, payload: string) => void,
+  voiceLang: ?string
 };
-const Message = ({ chat, onButtonClick }: MessageProps) => {
+
+const speak = (message, voiceLang) => {
+  let synth = window.speechSynthesis;
```

Comment:
Do you mind checkin, that this feature is actually supported by the browser. Most browsers support this API by now, but not all. (I am looking at you IE) Otherwise, it will crash the whole chat interface. Thanks